### PR TITLE
Docs/readme.md

### PR DESCRIPTION
**Pull request by mikeharrison57**
**[Project Ticket](https://github.com/lkarpins/fitlit-starter-kit/projects/1#card-82239261)**

**Describe the functionality to be tested:**

- Added README.md. Includes all relevant sections. It is a draft, so we can still make some minor revisions if we need to. 

**Type of Change**

- [x] README.md

**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new bugs

**Giphy of how you feel submitting this PR:**

[Happy Sunday!](https://media0.giphy.com/media/fCCCy9QuTtsFM9S88n/giphy.gif?cid=ecf05e4767d2420505bc7a44ba8c585d04188501f283fd97&rid=giphy.gif&ct=g)

### CODE REVIEW

**Review by Rallen13 — APPROVED** [review]
Great job Michael! The readme is very thorough. Thanks so much for taking that on. Happy Sunday, indeed!

File: README.md [hunk 1]
```diff
@@ -1,78 +1,91 @@
...
+![HealthHub Recording](https://user-images.githubusercontent.com/95496577/169706726-6dc58550-073f-4af8-89a8-c24313bcb7bc.gif)
+
+## Wireframe
+
+- [Figma Wireframe](https://www.figma.com/file/MAcq55QYBdenQxU3mBVcVz/FitLit-Wireframe?node-id=0%3A1) 
+
+<img width="721" alt="New Wire Frame Layout" src="https://user-images.githubusercontent.com/95496577/169707014-ea0f4135-dca2-4ea0-8bbe-2e4e7047f150.png">
+
+## Technologies Used
+
```
Comment (Rallen13) [inline README.md:32]: Maybe we should add Chart.js and Normalize.css to our technologies? Just a suggestion.

File: README.md [hunk 2]
```diff
@@ -1,78 +1,91 @@
...
+## Context and Features
+
+- HelathHub is a health and wellness application. Upon page load, the site displays a user’s information (name, address, and email), hydration data, and sleep data. It compares their daily step goal to an average amongst a database of users. The site also greets the user by their first name at the top of the web page. This application was created using Webpack, which is a modular bundler used to group HTML, CSS, and JavaScript files into more efficient files for a browser to read. It also accounts for npm packages, which our application includes many of. 
+
+- We made use of many packages to bring our application together including `Normalize.css` and `Charts.js`. The `Normalize.css` package allowed us to make our application render more consistently on different browsers. The `Charts.js` package allowed us to create charts that represent our weekly data for user hydration and sleep patterns. Both of these charts are dynamic and generated based on the data for a specific user. They are displayed in our HealthHub widgets. The widgets also display daily and average information for each respective data type in an appealing and eye catching manner.
```
Comment (Rallen13) [inline README.md:44]: It should be Chart.js instead of Charts.js :)

**Review by lkarpins — APPROVED** [review]
Wow, Michael! Awesome work! Great detail! 

Thank you for taking on the README.md!

**Pull request merged**